src/views/checkups/storage/details/CheckupsStorageDetailsPageSection.tsx

```diff
@@ -0,0 +1,216 @@
+            <DescriptionListGroup>
+              <DescriptionListTerm>
+                {t('VirtualMachine with non virt RBD StorageClass')}
+              </DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STORAGE_CHECKUPS_WITH_NON_RBD_STORAGE_CLASS] || t('None')}
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>
+                {t('VirtualMachine boot from golden image')}
+              </DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STORAGE_CHECKUPS_BOOT_GOLDEN_IMAGE] || t('None')}
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('VirtualMachine live migration')}</DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STORAGE_CHECKUP_LIVE_MIGRATION] || t('None')}
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Timeout')}</DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STORAGE_CHECKUP_TIMEOUT]}
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+          </DescriptionList>
+        </GridItem>
+        <GridItem span={6}>
+          <DescriptionList>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Namespace')}</DescriptionListTerm>
+              <DescriptionListDescription>
+                <ResourceLink
+                  groupVersionKind={modelToGroupVersionKind(NamespaceModel)}
+                  name={configMap?.metadata?.namespace}
+                />
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Failure reason')}</DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STATUS_FAILURE_REASON] || t('None')}
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Complete time')}</DescriptionListTerm>
+              <DescriptionListDescription>
+                <Timestamp
+                  timestamp={configMap?.data?.[STATUS_COMPILATION_TIME_STAMP] || NO_DATA_DASH}
+                />
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>
+                {t('Storage class with empty claimPropertySets')}
+              </DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STORAGE_CHECKUPS_WITH_CLAIM_PROPERTY_SETS] || t('None')}
```

Comment:
This is the wrong key for this item. I didn't see a constant for the property `status.result.storageProfilesWithEmptyClaimPropertySets`.

Reply:
thanks!